### ISSUE TITLE
Update multifamily_beta project with attic type tsv files

### ISSUE THREAD

**Issue by joseph-robertson** [NREL/OpenStudio-BuildStock#384]

**Comment by shorowit** [comment]
@joseph-robertson This project no longer exists. Is this issue still relevant?

**Comment by afontani** [comment]
Couple Questions:
- Is this a reference to finished/unfinished attics and if we can model them for Single-Family Detached, Single-Family Attached, and Multi-family measures? 
- Or, is it just about creating a housing characteristic that describes the attic types?
- Is it related to [resstock-estimation issue #112](https://github.com/NREL/resstock-estimation/issues/112)

**Comment by joseph-robertson** [comment]
@afontani

As of v2.2.0, `project_singlefamilydetached` had a tsv file `Geometry Attic Type.tsv`. The purpose of it was to sample either "Unfinished Attic" or "Finished Attic or Cathedral Ceilings" for the buildstock csv (even though an unfinished attic was always modeled).

The `project_multifamily_beta` project did not have this file (for SFD orSFA/MF buildings). This github issue was about including `Geometry Attic Type.tsv` in `project_multifamily_beta` (which has now evolved into `project_national`). Since the tsv does not assign any measure arguments, it is not required. Although I defer to @ejhw on whether we still want to include it.

**Comment by ejhw** [comment]
Yes, we should add attic type, at least for SFD. It is important for knowing which homes are eligible for blown-in attic floor insulation.
